app/scripts/templates/settings/avatar.mustache

```diff
@@ -1,17 +1,13 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-account-picture-header">{{#t}}Account picture{{/t}}</h1>
-  </header>
-
-  <section>
-    <p class="avatar-wrapper">
-    </p>
-
-    <div class="error"></div>
-    <div class="success"></div>
-
-    <ul class="links">
-      <li><a href="/settings/avatar/change" id="change-avatar">{{#t}}Change{{/t}}</a>
-    </ul>
-  </section>
+<div id="change-avatar" class="settings-unit">
+  <div class="settings-unit-stub">
+    <header class="settings-unit-summary">
+      <h2 class="settings-unit-title">{{#t}}Account Picture{{/t}}</h2>
```

Comment:
`Account Picture` vs `Communication preferences` vs `Delete account`?
Should be `Account picture` for consistency?